test/common.py

```diff
@@ -359,4 +360,115 @@ def setdata(self,data):
                 self.data[i] = -1*temp;
         print(self.data)
 
-            
\ No newline at end of file
+
+class RLE:
+    """
+    These class is been created in order to store the value or the matrix which
+    are been used to test the RLE module.
+    The matrix are from github repositories for the purpose of input for
```

Comment:
Which github repositories? There are plenty out there... How do I check that you haven't made a mistake in the matrixes?